docs/book/intro.md

```diff
@@ -1,3 +1,132 @@
 # zend-mvc-middleware
 
-This component provides ...
+This library provides the ability to dispatch middleware pipelines in place of
+controllers within zend-mvc.
+
+## Dispatching PSR-7 Middleware
+
+[PSR-7](http://www.php-fig.org/psr/psr-7/) defines interfaces for HTTP messages,
+and is now being adopted by many frameworks; Zend Framework itself offers a
+parallel microframework targeting PSR-7 with [Expressive](https://docs.zendframework.com/zend-expressive).
+What if you want to dispatch PSR-7 middleware from zend-mvc?
+
+zend-mvc currently uses [zend-http](https://github.com/zendframework/zend-http)
+for its HTTP transport layer, and the objects it defines are not compatible with
+PSR-7, meaning the basic MVC layer does not and cannot make use of PSR-7
+currently.
+
+However, starting with version 2.7.0, zend-mvc offers
```

Comment:
I think a separate headline which contains the version number 2.7.0 would help here. This breaks up the entire section into smaller parts and giving a better overview of the content.